source/assets/js/playground/console-utils.ts

```diff
@@ -99,6 +102,19 @@ export function displayForConsoleLog(
         ];
       }
     }
+    if (item.type === 'debug') {
+      try {
+        const color = new Color(item.message);
```

Comment:
Can you add a TODO here to support colors in nested data structures once  https://github.com/sass/sass/issues/3957 is implemented?